src/plugins/setup.rs

```diff
@@ -300,3 +321,9 @@ fn run_substep_schedule(world: &mut World) {
         world.run_schedule(SubstepSchedule);
     }
 }
+
+/// Runs the [`PostProcessCollisions`] schedule.
+fn run_post_process_collisions_schedule(world: &mut World) {
+    debug!("running PostProcessCollisions");
+    world.run_schedule(PostProcessCollisions);
+}
```

Comment:
This is bikeshedding now, but I'm in two minds about whether this function should be 
* `run_post_process_collisions_schedule` or
* `run_post_process_collisions`

Similarly, for the log, since we've dropped `Schedule` from the name, should it be
* `"running PostProcessCollisions"` or
* `"running PostProcessCollisions schedule"`

Reply:
I might slightly prefer `run_post_process_collisions` and `"running PostProcessCollisions"` for brevity, but I have no strong preference either way. It's not very user-facing so it doesn't really matter

Reply:
Actually no, maybe `run_post_process_collisions_schedule` to make it clearer that that system doesn't actually do the post-processing

Reply:
Yep, looks good 👍